powershell-module/README.md

```diff
@@ -17,6 +18,24 @@ The Az.Tools.Migration PowerShell module contains cmdlets that perform the follo
 1. Generate an upgrade plan to convert AzureRM module references to Az module commands.
 1. Execute the upgrade plan to modify your PowerShell codebase.
 
+## Getting started
+
+You can use either method to use this module:
+
+1. Install this module from the [PowerShell gallery](https://www.powershellgallery.com/packages/Az.Tools.Migration).
+
+```
+Install-Module -Name Az.Tools.Migration
+```
+
+1. Clone this repository and import the module locally.
```

Comment:
This didn't auto-bullet to 2 in the markdown preview, so you might want to specify “2.” on the beginning of the line instead.

Reply:
Thanks, I missed this.